macros/dmt_get_test_sql.sql

```diff
@@ -28,11 +28,26 @@
             {% set ns.test_sql = render(ns.test_sql)|replace(ns.rendered_keys[k], v) %}
         {% endfor %}
 
-        {% set mock_model_relation = dbt_datamocktool._get_model_to_mock(
+        {# Store model result for visibility in case of unit test failure #}
+        {% set identifier_name = model.name %}
+        {% if test_case %}
+            {% set identifier_name = identifier_name ~ '__test_case_'~test_case %}        
+        {% endif %}
+        {# Note: possible to hardcode desired database.schema to store the mocked model output #}
+        {% set mock_model_relation = api.Relation.create(
+                    database=model.name,
+                    schema=model.schema,
+                    identifier=identifier_name,
+                    type='view') %}
+        {# Create view to expose full definition #}
+        {% do run_query(create_view_as(relation, ns.test_sql)) %}
+
+        {# FIXME: commenting out as not sure best way to incorporate the dispatch pattern #}
+        {# {% set mock_model_relation = dbt_datamocktool._get_model_to_mock(

```

Comment:
@mjirv would appreciate your support here, as I am not sure what is the best way to proceed with the dispatch pattern
Should I change all cases to make them store the mocked model output as a view as well?